cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/StructuredTableAdmin.java

```diff
@@ -26,6 +30,24 @@
    * Create a StructuredTable using the {@link StructuredTableSpecification}.
    *
    * @param spec table specification
+   * @throws IOException if there is an error creating the table
+   */
+  void create(StructuredTableSpecification spec) throws IOException;
+
+  /**
+   * Get the {@link StructuredTableSpecification} corresponding to the given table id
+   *
+   * @param tableId the table id
+   * @return the specification for the table
+   */
+  @Nullable
+  StructuredTableSpecification getSpecification(StructuredTableId tableId);
+
+  /**
+   * Drop the StructuredTable
```

Comment:
will this be a synchronous operation? Could you add more information around that in javadoc?

Reply:
add period at end of sentence.

Reply:
Yes. This will be a synchronous operation. Added more to javadoc